src/utilities/levDist.js

```diff
@@ -1,5 +1,7 @@
 function levDist(s,t){//https://stackoverflow.com/a/11958496/5697837
 	// Step 1
+	s = s.replace("’", "'")
+	t = t.replace("’", "'")
```

Comment:
Could have been done by replacing *before* passing the strings to leveDist(), but ignoring similar characters is probably best anyway.

Reply:
Yes I thought so too but like you said I concluded that always ignoring these characters was the best way